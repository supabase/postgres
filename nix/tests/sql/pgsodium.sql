select
  status
from
  pgsodium.create_key();
