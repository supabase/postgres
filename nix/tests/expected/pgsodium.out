select
  status
from
  pgsodium.create_key();
 status 
--------
 valid
(1 row)

